### ISSUE TITLE
sensor is not respecting it's given name

### ISSUE THREAD

**Issue by pejotigrek** [PiotrMachowski/Home-Assistant-custom-components-Antistorm#13]
not a major bug, but my OCD can't unsee it ;)
as I'm always trying to set up names "my way" for sorting/filtering purposes, I gave new names to binary_sensor and sensor:

```
binary_sensor:
- platform: antistorm
  name: 0_wth
  station_id: 275
  monitored_conditions:
  - 'storm_alarm'
  - 'rain_alarm'
  - 'storm_active'

sensor:
- platform: antistorm
  name: 0_wth
  station_id: 275
  monitored_conditions:
  - 'storm_probability'
  - 'storm_time'
  - 'rain_probability'
  - 'rain_time'
```

_binary_sensor_ works fine and produces: `binary_sensor.0_wth_275_rain_alarm, binary_sensor.0_wth_275_storm_active, binary_sensor.0_wth_275_storm_alarm`, but _sensor_ still creates it's own `sensor.antistorm_275_rain_probability, sensor.antistorm_275_rain_time, sensor.antistorm_275_rain_time, sensor.antistorm_275_storm_time`.

**Comment by PiotrMachowski** [comment]
Absolutely understandable :D

**Comment by pejotigrek** [comment]
I'm thinking right now that it might be a breaking change for some people ;)

**Comment by PiotrMachowski** [comment]
It won't if I will add a config flow ;)

**Comment by PiotrMachowski** [comment]
Fixed in v2.0.0
